test/atlabs/SMS.test.js

```diff
@@ -0,0 +1,104 @@
+const chai = require("chai");
+const nock = require("nock");
+
+const SMS = require("../../lib/atlabs/SMS");
+
+const scope = nock("https://sandboxapi.fsi.ng");
```

Comment:
Use random URL for nock `i.e. faker.internet.url()`

Reply:
Done.